cluster/ci/config/prow/plugins.yaml

```diff
@@ -920,6 +920,16 @@ config_updater:
     cluster/ci/config/mirroring/knative/knative_*:
       name: image-mirror
 
+    cluster/ci/monitoring/datasources.yaml:
```

Comment:
Sorry, what I meant is that changes to _this_ file need to land in a separate PR, first